images/win/scripts/Installers/Install-PyPy.ps1

```diff
@@ -82,6 +83,12 @@ $toolsetVersions = Get-ToolsetContent | Select-Object -ExpandProperty toolcache
 # Get PyPy releases
 $pypyVersions = Invoke-RestMethod https://downloads.python.org/pypy/versions.json
 
+# required for html parsing
+Install-Module PowerHTML -Scope CurrentUser 
+Import-Module PowerHTML
+$checksums = (Invoke-RestMethod -Uri 'https://www.pypy.org/checksums.html' | ConvertFrom-HTML).SelectNodes('//*[@id="content"]/article/div/pre')

```

Comment:
This looks extremely fragile. Do we really need to parse HTML? Why not to treat page as plain text and search for proper "checksum-filename" pair instead?

Reply:
I do not agree with you. Page is html, treating it as plain text is more fragile from my point of view

Reply:
that's how parsing works. if that has changed, image generation will fail. that is expected.
I'm not writing uber parser that will survive such changes, I think it is fine